src/nwbinspector/checks/ecephys.py

```diff
@@ -79,3 +79,16 @@ def check_spike_times_not_in_unobserved_interval(units_table: Units, nunits: int
                     "observed intervals."
                 )
             )
+
+
+@register_check(importance=Importance.CRITICAL, neurodata_type=ElectricalSeries)
+def check_electrical_series_dtype(electrical_series: ElectricalSeries):
```

Comment:
```suggestion
def check_electrical_series_conversion_factors(electrical_series: ElectricalSeries):
```

Since we're not just testing the the dtype, suggesting a more descriptive name; if you have any better ideas for a name feel free to suggest those as well

Reply:
Thank you, makes more sense